src/api/aux.cpp

```diff
@@ -30,9 +33,9 @@ void parse_json_file(std::vector<Token> &tokens, std::vector<Order> &orders, std
             t = static_cast<boost::multiprecision::mpf_float>(tt);
         }
         if (i.value()["alias"].is_null())
-            tokens.push_back(Token(i.key(), "_NULL", i.value()["decimals"], t));       // TODO: Check if multiple tokens with "null" alias can create any issue
+            tokens.push_back(Token(i.key(), "_NULL", i.value()["decimals"], t, normalize_priority));       // TODO: Check if multiple tokens with "null" alias can create any issue
         else
-            tokens.push_back(Token(i.key(), i.value()["alias"], i.value()["decimals"], t));
+            tokens.push_back(Token(i.key(), i.value()["alias"], i.value()["decimals"], t, normalize_priority));
```

Comment:
On an if-then-else, it is good practice (and usually used by the compiler) to write the more likely branch first.